### ISSUE TITLE
Image alt text on mobile, unable to submit

### ISSUE THREAD

**Issue by hardillb** [mastodon/mastodon#32198]
### Steps to reproduce the problem

1. Attach image to a toot from the web interface on a mobile phone
2. Add a long image description
3. Submit button is not accessible
...

### Expected behaviour

modal should be scrollable to allow access to button

### Actual behaviour

submit button pushed off screen and no way to reach it to submit text

### Detailed description

(will attach screenshot) 
![Screenshot_20241001-133756](https://github.com/user-attachments/assets/a6426032-e5eb-423b-abd3-472174d95dad)

### Mastodon instance

bluetoot.hardill.me.uk

### Mastodon version

v4.2.10

### Browser name and version

Chrome

### Operating system

Android 14

**Comment by ClearlyClaire** [comment]
I'm surprised this hasn't kicked to the vertical layout. In any case, I think this is addressed in 4.3, thanks to #27095
